src/fondant/cli.py

```diff
@@ -228,6 +234,7 @@ def compile(args):
             pipeline=args.pipeline,
             extra_volumes=args.extra_volumes,
             output_path=args.output_path,
+            build_args=args.build_arg,
```

Comment:
Any reason we have build_args (plural) here?

Reply:
There can be multiple build arguments, but the command line is `--build-arg` since you provide them one at a time.

Eg. `--build-arg FONDANT_VERSION=a8b9fd --build-arg CUSTOM_ARG=value` becomes `build_args = ["FONANT_VERSION=a8b9fd", "CUSTOM_ARG=value"]`